.gitignore

```diff
@@ -7,4 +7,4 @@ test/fixtures/underscore
 test/*.js
 examples/beautiful_code/parse.coffee
 *.gem
-/node_modules
+/node_modules
```

Comment:
Files should always end with a newline. Also, please review your changes before you commit, to avoid pointless changes like this.